docs/TrafficSplitting.md

```diff
@@ -0,0 +1,157 @@
+# Traffic Splitting
+
+The purpose of this section of the kn documentation is to describe the command verbs for traffic splitting operations and present some examples.
+
+`kn` client can work with following properties of a traffic target defined in traffic block of a service spec.
+ - **RevisionName**: RevisionName of a specific revision to which to send a portion of traffic.
+ - **Tag**: Tag is optionally used to expose a dedicated url for referencing this target exclusively.
+ - **Percent**: Percent specifies percent of the traffic to this Revision.
+
+## Operations
+
+Operations required to perform traffic splitting use cases, for example: blue/green deployment, etc can be summarized as below:
+ - Tag targets
```

Comment:
I would have proposed names for these and small explanation if possible. Since one goal here is to see what other operations people reading this spec can come up with...